### PR TITLE
add values

### PR DESCRIPTION
**Pull request by calvinmetcalf**
you can specify values in your prepared statements, which is what this does

### CODE REVIEW

File: lib/string.js [hunk 1]
```diff
@@ -150,9 +150,13 @@ exports.format = function (sql, values, timeZone, method) {
   }
 
   if (method === 'insert') {
+<<<<<<< HEAD
```
Comment (knownasilya) [inline lib/string.js]: Probably didn't want this here

Comment (calvinmetcalf) [reply lib/string.js]: already fixed, try refreshing

**Comment by calvinmetcalf** [comment]
ok @knownasilya think this is ok to merge ?

**Comment by knownasilya** [comment]
Don't remember what this was about, have an example query?

**Comment by calvinmetcalf** [comment]
I have no memory of this either, but a fix to the batch operations is built on top of this and I've tested it locally and it works

**Comment by calvinmetcalf** [comment]
ok I think this adds parameters to the prepared statements

**Comment by knownasilya** [comment]
looks good to me lol

**Pull request merged**